docs/octopus-cloud/index.md

```diff
@@ -42,6 +42,14 @@ You will be taken to the account provisioning screen. Please note it can take fi
 
 When the instance is ready, you will see it (and any other instances you have access to) the next time you log in to your Octopus account at [https://account.octopus.com/account/signin](https://account.octopus.com/account/signin).
 
+## Uploading packages to Octopus Cloud Servers
+
+Factors such as geopgrahical distance, network bandwidth, and network congestion may cause package uploads to your Octopus Cloud Server to take longer than expected.
+
+If you are having difficulty uploading packages to your Octopus built-in package feed within the default timeout threshold - which is usually within 5 minutes, you might be affected by one or more of the factors mentioned above. In this case, you could try increasing the timeout threshold of `octo.exe` or `nuget.exe` to a value that suits you. We also recommend using `octo.exe`, `Octopus.Client`, or the `TeamCity Plugin (v4.41.0+)` because they have the advantage of utilizing delta compression.
+
+In the near future, the Octopus Cloud will become available in additional regions such as Sydney, London, and Frankfurt and you'll have the ability to move between regions.
```

Comment:
Hi Lawrence, the addition looks good.
I just have one concern. Rather than providing a list of possible regions, can we just say: 
```suggestion
In the near future, the Octopus Cloud will become available in additional regions and you'll have the ability to move between regions.
```
That way if anything changes between now and the new regions being rolled out we have promised any specific centers.